build/cdc_services/run.sh

```diff
@@ -88,7 +88,7 @@ else
 fi
 
 # Wait for any process to exit
-wait
+wait -n
```

Comment:
In the happy case, we want this to block until all services have started. Checking if this continues to be the case after this change?

Reply:
Confirmed! The behavior of this script is the same in the happy case - confirmed by using a bind mount with datacommons-services:stable to replace run.sh with the new version and comparing the command output to the output without the bind mount.

Reply:
Thank you!